### PR TITLE
Fix l10n it withholding tax date maturity

### PR DESCRIPTION
**Pull request by alessandrocamilli**
In the withholding tax move is now possible to view and change date maturity.
In the head of withholding tax move there is partner's name and wt's name.

### CODE REVIEW

File: l10n_it_withholding_tax/models/withholding_tax.py [hunk 1]
```diff
@@ -189,6 +189,16 @@ class withholding_tax_move(models.Model):
...
     account_move_id = fields.Many2one('account.move', 'Account Move',
                                       ondelete='cascade')
+    display_name = fields.Char(
+        string='Name', compute='_compute_display_name',
+    )
+
+    @api.one
+    def _compute_display_name(self):
+        name = '%s - %s' % (self.partner_id.name,
+                            self.withholding_tax_id and
```
Comment (OpenCode) [inline l10n_it_withholding_tax/models/withholding_tax.py]: Can you use inline if: 
self.withholding_tax_id.name if self.withholding_tax_id else ''
?

File: l10n_it_withholding_tax/models/withholding_tax.py [hunk 2]
```diff
@@ -189,6 +189,16 @@ class withholding_tax_move(models.Model):
     date_maturity = fields.Date('Date Maturity')
     account_move_id = fields.Many2one('account.move', 'Account Move',
                                       ondelete='cascade')
+    display_name = fields.Char(
+        string='Name', compute='_compute_display_name',
+    )
+
+    @api.one
```
Comment (eLBati) [inline l10n_it_withholding_tax/models/withholding_tax.py]: @alessandrocamilli dovresti usare api.multi e ciclare su self

Comment (alessandrocamilli) [inline l10n_it_withholding_tax/models/withholding_tax.py]: @eLBati fatto! Ho aggiunto anche un paio di features.

File: l10n_it_withholding_tax/views/withholding_tax.xml [hunk 3]
```diff
@@ -109,9 +109,12 @@
 	            	<group>
 	            		<group>
 	            			<field name="partner_id"/>
-	            			<field name="date"/>
+	            			<field name="withholding_tax_id"/>
+	            			<field name="base"/>
+	            			<field name="tax"/>
```
Comment (eLBati) [inline l10n_it_withholding_tax/views/withholding_tax.xml]: @alessandrocamilli 
```Use wrong tabs indentation instead of four spaces```

Comment (alessandrocamilli) [inline l10n_it_withholding_tax/views/withholding_tax.xml]: @eLBati done

**Pull request merged**